homeassistant/components/netgear/sensor.py

```diff
@@ -78,6 +78,6 @@ def async_update_device(self) -> None:
         self._device = self._router.devices[self._mac]
         self._active = self._device["active"]
         if self._device[self._attribute] is not None:
-            self._state = self._device[self._attribute]
+            self._state = self._device.get(self._attribute)
```

Comment:
We don't need to use dict.get here since we don't do it above. If the key can be missing we need to handle it already on the line above.

Reply:
Damn, how could I have missed that.
Thank you for pointing it out!
I just changed it.